app/forms/idv/api_image_upload_form.rb

```diff
@@ -204,14 +209,15 @@ def as_readable(image_key)
       end
     end
 
-    def update_analytics(client_response)
+    def update_analytics(client_response, vendor_request_time_in_ms)
```

Comment:
I would consider making these keyword arguments, less likely to order them incorrectly
```suggestion
    def update_analytics(client_response:, vendor_request_time_in_ms:)
```